src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const paymentPerDay = 40;
+  const threeDaysDiscount = 20;
+  const weekDiscount = 50;
+  const calcPayment = days * paymentPerDay;
+
+  if (days >= 7) {
```

Comment:
For days: `7` and `3` create separate values, as well. 
For example: 
```
const shortPeriodDays = 3
const longPeriodDays = 7
```

Better to show in what units you measure the period. 👆